src/styles/index.css

```diff
@@ -0,0 +1,128 @@
+@import './reset.css';
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  max-width: 200px;
+  box-sizing: border-box;
+}
+
+.card__content {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
```

Comment:
The `width` property is not necessary here if you already set the `max-width` property in the `.card` class. It's better to remove it to avoid confusion.